content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -150,27 +151,13 @@ In this step, you use CMake to build either a non-monolithic or a monolithic pro
 
 ### Source engine
 
-With a source engine, you must use a project-centric configuration to create a project game release layout. In a project-centric configuration, the CMake source directory is in the project build directory. So O3DE tools (for example, **O3DE Editor** and Asset Processor) are located in your `<project>\build` directory.
+With a source engine, you must use a project-centric configuration to create a project game release layout. In a project-centric configuration, the CMake source directory is in the project build directory. So O3DE executables (for example, **O3DE Editor**, Asset Processor, and Game Launcher) are located in your `<project>\build` directory.
 
-To create a non-monolithic or monolithic project game release layout using a source engine, complete the following steps.
+Now, you will choose a build type for your project game release layout.
 
-#### Option 1: Non-monolithic
+#### Option 1: Monolithic (Game only)
 
-A source engine supports non-monolithic projects by default. As detailed in [Prerequisites](#prerequisites), you should have already created a Visual Studio project for your project and registered your project to your engine.
-
-To build your non-monolithic project for release, use CMake to invoke Visual Studio builder. Specify the `INSTALL` target and `release` configuration.
-
-```cmd
-cd C:\MyProject
-cmake --build build\windows --target INSTALL --config release 
-```
-
-You can specify a particular non-monolithic build by appending the option `-DLY_MONOLITHIC_GAME=0`. This command generates O3DE tools (such as Editor, Asset Processor, and Game Launcher) and dependent `.dll` files. It also bundles your project's product assets that are located in `<project>\Cache\pc` into an `engine.pak` file.
-
-The result is a project game release layout in the install directory that's located at `<install>\bin\Windows\release\Default`. In your project game release layout, your `engine.pak` file is located in `Cache\pc`.
-
-
-#### Option 2: Monolithic
+The most common workflow is to release your game only, in which case you should choose the **monolithic** build. This will build only your game/server launcher and any necessary dependencies.
```

Comment:
There's a display bug with `{{< note >}}` shortcodes inside of `{{< tabs >}}`. It's causing the Note callout to outdent and the steps below to both be labelled **1.**. Would you mind addressing this while you're here? As a workaround, I'd suggest simply removing the `{{< note >}}` shortcode around the text below that begins "Use `Visual Studio 16` as the generator". You could start it with "**Note:** " instead. Then just make sure that paragraph is indented.

Reply:
Thanks I didn't notice that! Yep I can do as you suggested here and below!